package.json

```diff
@@ -1,20 +1,55 @@
 {
-  "name": "storyblok-vue",
-  "version": "1.0.5",
-  "description": "Storyblok directive for get editable elements.",
-  "main": "storyblok-vue.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/storyblok/storyblok-vue"
-  },
-  "keywords": [
-    "vue",
-    "storyblok"
+  "name": "@storyblok/vue-ws",
+  "private": true,
+  "version": "0.0.1",
```

Comment:
@alexjoverm has any value this version here?

Reply:
Not at all, it's the workspace root so it just needs a name and a version to work